doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -3,18 +3,94 @@
+These instructions are designed to help users get started with a single-locale
+(shared-memory) implementation of Chapel as quickly as possible. If you want to
+understand Chapel's configuration options, build process, and installation more
+thoroughly, please refer to :ref:`readme-chplenv` and :ref:`readme-building`
+instead.
 
-These instructions first have you build a minimal, low-performance
-configuration of Chapel to reduce build times and the potential for
-third-party portability issues.  Once you are interested in getting
-better performance or using a full-featured version of Chapel, refer
-to :ref:`using-a-more-full-featured-chapel` below.
+The options for getting started, in order of preference, are:
+
+1) `Binary Release via a Package Manager`_
+2) `Containers (Docker)`_
+3) `From Source Builds via Spack`_
+4) `From Source`_
+
+Binary Release via a Package Manager
+------------------------------------
+
+Chapel provides binary releases for several platforms and operating systems.
+If you are using a platform for which Chapel provides a binary release, we
+recommend using it to get the easiest and fastest start with Chapel. To install Chapel this way, download the appropriate package for your OS and platform and install it using your system's package manager.
+
+For example, on Ubuntu 24 with x86_64, you can use the following to install Chapel 2.4:
+
+.. code-block:: bash
+   wget https://github.com/chapel-lang/chapel/releases/download/2.4.0/chapel-2.4.0-1.ubuntu24.amd64.deb
+   sudo apt-get update
+   sudo agt-get install ./chapel-2.4.0-1.ubuntu24.amd64.deb
+
+For example, on Fedora 41 with aarch64, you can use the following to install Chapel 2.4:
+
+.. code-block:: bash
+   wget https://github.com/chapel-lang/chapel/releases/download/2.4.0/chapel-2.4.0-1.fc41.aarch64.rpm
+   sudo dnf update
+   sudo dnf install ./chapel-2.4.0-1.fc41.aarch64.rpm
+
+See `this page <https://chapel-lang.org/download/#linux>`_ for the most up-to-date
+list of supported OSes and platforms.
+
+brew
+~~~~
+
+If you are using `brew <https://brew.sh/>`_, you can also install Chapel directly with ``brew install chapel``. This will install the latest Chapel release available in the brew repository.
+
+Containers (Docker)
+-------------------
+
+Chapel provides a number of Docker images that can be used to run Chapel in a containerized environment. This is a good option if you want to try Chapel without installing it on your host system or if you want to ensure that you have a consistent environment across different machines.
+
+We currently provide three different Docker images: chapel, chapel-gasnet, and chapel-gasnet-smp. The `chapel` image is the most basic and includes the Chapel compiler and standard library. The `chapel-gasnet` image includes support for multi-locale (distributed memory) programming using GASNet over ethernet, while the `chapel-gasnet-smp` image includes support for multi-locale programming using shared memory.
```

Comment:
I think this is fine as is, but it could also be converted into list form to make it a little easier to read